ambry-frontend/src/main/java/com/github/ambry/frontend/FrontendRestRequestService.java

```diff
@@ -74,7 +74,9 @@ class FrontendRestRequestService implements RestRequestService {
   private SecurityService securityService = null;
   private GetPeersHandler getPeersHandler;
   private GetSignedUrlHandler getSignedUrlHandler;
-  private NamedBlobListHandler listNamedBlobsHandler;
+  private NamedBlobListHandler namedBlobListHandler;
+  private NamedBlobGetHandler namedBlobGetHandler;
+  private NamedBlobDeleteHandler namedBlobDeleteHandler;
   private GetBlobHandler getBlobHandler;
   private PostBlobHandler postBlobHandler;
   private NamedBlobPutHandler namedBlobPutHandler;
```

Comment:
nit, for better readability, the 'NamedBlobPutHandler' (line 82) can be moved to be right after 'NamedBlobDeleteHandler' (line 79) so that the four NamedBlob handlers are together.